### DISCUSSION TITLE
[MonthPickerInput] Add excludeMonth

### DISCUSSION THREAD

**Discussion by davidchalifoux** [mantinedev/mantine#6262] [Feature requests]
The `DatePicker` component has the `excludeDate` param. It would be great if this same idea was available on the `MonthPicker` component.
I'm thinking `excludeMonth` where you can pass a function to determine if a month is disabled.

**Comment by rtivital** [comment]
You can do that with `getMonthControlProps` prop.

**Reply by davidchalifoux** [reply]
That's perfect, I missed that option in the docs. Thank you as always Vitaly!
